library_builders.sh

```diff
@@ -203,6 +211,43 @@ function build_blosc {
     touch blosc-stamp
 }
 
+function build_blosc2 {
+    if [ -e blosc-stamp2 ]; then return; fi
+
+    curl -sLo c-blosc2-v2.7.1.tar.gz \
```

Comment:
@franzpoeschel Is there a reason you selected 2.7.1 and not the latest 2.9.3 for c-blosc2?

Reply:
Oh, I see:
```
  -- Using ZSTD internal sources.
      "STREQUAL" "arm64"
```

wild!

Reply:
I recall picking the most recent version that I saw. So, no reason probably, no